_docs/style/colors.md

```diff
@@ -11,20 +11,142 @@ status: complete
 {% include toc.html %}
 
 <section class="static-section"  markdown="1">
-### Primary Colors
 
-Primary colors should be used sparingly for primary and secondary actions,
-alerts, or data visualizations.
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## The Helix color palette
+
+The Helix color palette consists of deep modern hues juxtaposed with simple monochromatic environments. Our color palette has been selected to enhance usability while adhering to accessibility criteria. Guidelines for color usage are outlined for individual components.
+
+{% figure [caption:"Helix Color Palette"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/style/color-palette/color-palette.png" width="1900"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Guidelines
+
+We have designed the Helix color palette very intentionally so that it:
+
+- **Communicates meaning**: We use color to communicate meaning so that the user can make sense of all of the data presented to them.
+- **Draws the attention of the user**: We use color to draw either more or less of the user's attention to areas of the page that require attention.
+- **Orients the user**: We use color to enhance way-finding to that user don't become confused about location.
+
+Any deviation from the predefined colors introduces an inconsistency that can be confusing to the user. Strict adherence to the color palette and its families is important so that we can successfully communicate meaning, draw attention, and orient the user.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you work with the color palette:
+
+- Do not add colors to the color palette or to color families.
+- Be intentional about using color families. Don't use colors outside of a color family for a component that belongs to the family.
+- When you apply a color to an object, consider all of the other colors used on the page. Because color commands the attention of the user, ensure that the colors you use command the right amount of attention.
```

Comment:
ensure that the colors you use get the right amount of attention.

Reply:
Done.